### PR TITLE
Allow configuring target species that are always predicted

### PR DESCRIPTION
**Pull request by timmh**
This PR adds the option to specify a set of species ("target species") for which the classifier will always output the raw classification scores (logits), irrespective of whether they are within the top-scoring set or not. Specifically, this PR:

- adds a commandline option in `speciesnet/scripts/run_model.py` to specify a text file containing target species in the same format as the existing taxonomy.txt files
- adapts SpeciesNetClassifier to parse this file (if present) and always add logits for those target classes to the output
- adapts output documentation in the README accordingly

### CODE REVIEW

**Comment by agentmorris** [comment]
Thanks for your contribution, we're really interested to see all the things you're cooking up with the work you're doing around SpeciesNet!
